f3-sidecar/run.go

```diff
@@ -72,7 +74,13 @@ func run(ctx context.Context, rpcEndpoint string, jwt string, f3RpcEndpoint stri
 		logger.Warn("InitialPowerTable is undefined")
 		m.InitialPowerTable = cid.Undef
 	}
-	m.NetworkName = gpbft.NetworkName(network)
+	// Use "filecoin" as the network name on mainnet, otherwise use the network name. Yes,
```

Comment:
https://github.com/filecoin-project/lotus/blob/v1.32.3/chain/lf3/config.go#L83

```go
// Use "filecoin" as the network name on mainnet, otherwise use the network name. Yes,
	// mainnet is called testnetnet in state.
	if nn == "testnetnet" {
		nn = "filecoin"
	}
```